indico/modules/events/registration/models/registrations.py

```diff
@@ -462,7 +462,10 @@ def display_regform_url(self):
     @property
     def is_ticket_blocked(self):
         """Check whether the ticket is blocked by a plugin."""
-        return any(values_from_signal(signals.event.is_ticket_blocked.send(self), single_value=True))
```

Comment:
I don't think changing this here is correct - the property is supposed to check whether *tickets* (ie templates that have the `is_ticket` flag) are blocked.

It might also break cases where you try to print a ticket form the designer view (that one does not use the default template)